docs/front-end/sustainability.md

```diff
@@ -0,0 +1,7 @@
+# Sustainability
+
+In order to make the site more sustainable we ensure that images are served in a suitable size for the screen they are displayed on, whether they are background images or images loaded with an <img> tag. We can use <picture> elements with srcset and sizes. See https://docs.wagtail.org/en/v5.2.2/advanced_topics/images/image_file_formats.html#using-the-picture-element.
```

Comment:
Suggestion: Is it worth mentioning that images should be in `webp` format?

Reply:
good catch - I have added that in.